docs/start_guide.md

```diff
@@ -0,0 +1,279 @@
+# Getting Started
+
+(reword this)
+This is a thorough start guide to show you each detail of Atomic App.
+
+## Basic commands
+
+The __four__ basic commands of atomicapp are:
+
+__atomicapp fetch__: Retrieving a packaged container and exporting it to a directory.
+
+ex. `atomicapp fetch projectatomic/helloapache`
+
+__atomicapp run__: Running a packaged container on a specified provider. Unless a directory is specified, `run` will also perform `fetch`.
+
+ex. `atomicapp run projectatomic/helloapache --provider=kubernetes`
+
+__atomicapp stop__: Stopping a deployed Nulecule on a specified provider. Whether you're using Kubernetes, OpenShift or Docker, Atomic App will stop the containers.
+
+ex. `atomicapp stop ./myappdir --provider=kubernetes`
+
+__atomicapp genanswers__: By examing the `Nulecule` file. Atomic App will generate an `answers.conf` file to be used for non-interactive deployment.
+
+ex. `atomicapp genanswers ./myappdir`
+
+For more detailed information as well as a list of all parameters, use `atomicapp --help` on the command line. Alternatively, you can read our [CLI doc](docs/cli.md).
+
+## Atomic App on Project Atomic hosts
+
+If you are on a [Project Atomic host](https://projectatomic.io/download) you can interact with `atomicapp` via the `atomic` cli command.
+
+Some commands for `atomicapp` on an atomic host are a bit different.
+
+However. Regardsless of the `atomic run` command, a `--mode` can be passed to change the functionality of the command. 
+
+`atomicapp fetch projectatomic/helloapache`
+
+vs
+
+`atomic run projectatomic/helloapache --mode fetch`
+
+`atomicapp run projectatomic/helloapache`
+
+vs
+
+`atomic run projectatomic/helloapache`
+
+`atomicapp stop ./myappdir`
+
+vs
+
+`atomic stop projectatomic/helloapache ./myappdir`
+
+`atomicapp genanswers ./myappdir`
+
+vs
+
+`atomic run projectatomic/helloapache ./myappdir --mode genanswers`
+
+## Building your first Atomic App
+
+A typical Atomic App or "Nulecule" container consists of the following files:
+```sh
+~/helloapache
+▶ tree
+.
+├── answers.conf.sample
+├── artifacts
+│   ├── docker
+│   │   └── hello-apache-pod_run
+│   ├── kubernetes
+│   │   └── hello-apache-pod.json
+│   └── marathon
+│       └── helloapache.json
+├── Dockerfile
+├── Nulecule
+└── README.md
+```
+
+We will go through each file and folder as we build our first Atomic App container.
+
+For this example, we will be using the [helloapache](https://github.com/projectatomic/nulecule-library/tree/master/helloapache) example from the [nulecule-library](https://github.com/projectatomic/nulecule-library) repo.
+
+In order to follow along, fetch the container and `cd` into the directory:
+```sh
+atomicapp fetch --destination helloapache projectatomic/helloapache
```

Comment:
still would be nice to fetch into a directory that was named a little more appropriately to indicate it is a directory:

localdir or something like that